src/databricks/labs/ucx/workspace_access/redash.py

```diff
@@ -82,13 +83,9 @@ def get_apply_task(self, item: Permissions, migration_state: MigrationState):
     def _safe_get_dbsql_permissions(self, object_type: sql.ObjectTypePlural, object_id: str) -> sql.GetResponse | None:
         try:
             return self._ws.dbsql_permissions.get(object_type, object_id)
-        except DatabricksError as e:
-            if e.error_code in ["RESOURCE_DOES_NOT_EXIST", "RESOURCE_NOT_FOUND", "PERMISSION_DENIED"]:
-                logger.warning(f"Could not get permissions for {object_type} {object_id} due to {e.error_code}")
-                return None
-            else:
-                msg = f"{e.error_code} can be retried, doing another attempt..."
-                raise RetryableError(message=msg) from e
+        except NotFound:
+            logger.warning(f"removed on backend: {object_type} {object_id}")
```

Comment:
It's common practice to include the caught exception message into log along with context as this quickens the debug cycle.